test/src/main/java/org/apache/accumulo/test/functional/YieldingIterator.java

```diff
@@ -95,8 +95,8 @@ public void next() throws IOException {
    */
   @Override
   public Value getTopValue() {
-    String value = Integer.toString(yieldNexts.get()) + ',' + Integer.toString(yieldSeeks.get())
-        + ',' + Integer.toString(rebuilds.get());
+    String value =
+        Integer.toString(yieldNexts.get()) + ',' + yieldSeeks.get() + ',' + rebuilds.get();
```

Comment:
```suggestion
        yieldNexts.get() + "," + yieldSeeks.get() + "," + rebuilds.get();
```
Could do this if we wanted to avoid using the first `Integer.toString` as well. Either way, LGTM.

Reply:
Fixed in 4df7b130fc